Libraries/libutil/Sources/Relative.cpp

```diff
@@ -106,7 +140,24 @@ normalized() const
         output += (Traits::IsSeparator(_raw[n]) ? Traits::Separator : _raw[n]);
     }
 
-    output += NormalizePath(_raw, _raw.begin() + start, _raw.end(), absolute, true, Traits::Separator, &Traits::IsSeparator);
+    std::vector<std::string> relativeOutput = NormalizePathComponents(
```

Comment:
Why does `normalized` not just call `normalizedComponents`? It looks like the functions are duplicates of each other aside from the loop on lines 153-160 and the conversion on lines 122-123.

Reply:
I just wanted to minimize changes to the code, but that doesn't make sense now that you point this duplication out.